build_scripts/docker/docker.sh

```diff
@@ -75,3 +77,10 @@ build_docker_image_tgz() {
 move_package_to_dist_dir() {
     mv "$1" "$2/"
 }
+
+clean_stale_images() {
+   echo "Cleaning images"
+
+   sudo docker rmi $1
+   sudo docker image prune --force
```

Comment:
Does this remove the `bitnami/python:3.7` image? It would be best to keep it

Reply:
Nope, It removes the monkey-island image all other dangling images `None:None` that are coming out of the build.